### ISSUE TITLE
Adding hyperparameter information to trained model name helps in differentiate

### ISSUE THREAD

**Issue by venkanna37** [venkanna37/Label-Pixels#5]
Currently, patch size needs to be added.

**Comment by venkanna37** [comment]
for now, the model name, patch size, epochs and date are added to the model file. The issue will be open again if any information has to add to the model name.
